nick-fury/check_availability.py

```diff
@@ -0,0 +1,27 @@
+import sys
+import MySQLdb
+
+
+con1 = MySQLdb.connect(host='localhost',
+							user='root',
+							passwd="ankit@02217",
+							db='realtimebikepoint')
+cur1 = con1.cursor()
+
+
+def strip(s):
+	return s.strip().lower()
+
+argument = " ".join(sys.argv[1:])

```

Comment:
You should place this inside a `if __name__ == "__main__"` block - makes it clearer that this is supposed to be run separately

Reply:
I have resolved this issue.